src/components/Dashboard/FilingHistoryList/FutureEffective.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div v-if="filing" class="future-effective-details body-2">
+    <h4>{{_.subtitle}}</h4>
+
+    <p>The {{_.filingLabel}} date and time for {{_.companyLabel}}
+      will be <strong>{{effectiveDateTime}} Pacific Time</strong>.</p>
+
+    <p>If you wish to change the information in this {{_.filingLabel}}, you must
+      contact Registry Staff to file a withdrawal.</p>
+
+    <p>Withdrawing this {{_.filingTitle}} will remove this {{_.filingLabel}}
+      and all associated information, and will incur a $20.00 fee.</p>
+
+    <p v-if="filing.courtOrderNumber">Court Order Number: {{filing.courtOrderNumber}}</p>
+
+    <p v-if="filing.isArrangement">Pursuant to a Plan of Arrangement</p>
+
+    <p>Registries contact information:</p>
+
+    <contact-info class="pt-3" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+import { ContactInfo } from '@/components/common'
+import { HistoryItemIF } from '@/interfaces'
+
+@Component({
+  computed: { ...mapState(['entityName']) },
```

Comment:
I've seen a few instances where we've mapped this state to the component, then declared on the component as `readonly`. 
Would this be better as a getter? 
I believe if it is a Getter and it's ONLY used in the template, you don't need to declare it on the component, which is just a bonus if you don't need to use it within the `<script>`

Reply:
1. In this component, it is used in the script.
2. In other components it is used in the template only, but still think it's good practice to declare its reference (the readonly bit).
3. TBH, it would be even better to use a store getter for this property, but that's not the pattern in this UI.

Can I leave this?

Reply:
Of course!